src/textual/widget.py

```diff
@@ -393,6 +393,15 @@ def __init__(
     border_subtitle: str | Text | None = _BorderTitle()  # type: ignore
     """A title to show in the bottom border (if there is one)."""
 
+    @property
+    def is_mounted(self) -> bool:
```

Comment:
I see it's draft, but a wee FYI... we have the convention[^1] of: if it's a `@property`, document it as if its a property, so no `Returns` etc.

[^1]: Yes, yes, I know, right next to where you put this there's a couple of properties that violate that guideline; they're not all cleaned up from the beforetimes. :-P

Reply:
Haha, I actually almost didn't include this but second-guessed myself!